src/dialog-user/components/dialog-user/dialogField.ts

```diff
@@ -49,6 +50,7 @@ export class DialogFieldController {
   public setup() {
     this.clonedDialogField = _.cloneDeep(this.field);
     this.dialogField = this.service.setupField(this.field);
+    this.dialogField.disabled = this.reconfigureMode && !this.dialogField.reconfigurable;
```

Comment:
Can a field be disabled from automate?

I think this bit should only set disabled=true, but not overwrite with false if already present.

Reply:
Fixed (as discussed privately).